install-armv6-cross-toolchain

```diff
@@ -0,0 +1,163 @@
+#! /bin/bash
+
+set -ex
+
+# To provide support for Raspberry Pi Zero W a toolchain tuned for ARMv6 architecture must be used.
+# https://northerntech.atlassian.net/browse/MEN-2399
+# ARMV6_TOOLCHAIN_ROOT="/armv6-eabihf--glibc--stable-2020.08-1"
+TOOLCHAIN_PREFIX=/opt/
+ARMV6_TOOLCHAIN_ROOT="${ARMV6_TOOLCHAIN_ROOT:?No toolchain set}"
+RASPBIAN_VERSION="${VERSION:?No Raspbian version set}"
```

Comment:
Using spaces here makes me nervous. I also think it might hide the message you're trying to convey here ("No" not found). Underscores?

Reply:
Hmm, I think you misunderstand, this is quite common usage:

https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_06_02

Reply:
Ah, question mark! You're right, I misread that. (And a much better approach)